source/administration/command-line-tools.rst

```diff
@@ -902,11 +902,11 @@ mattermost extract-documents-content
 -------------------------------------
 
   Description
-    Extracts and indexes the contents of files shared prior to upgrading to Mattermost Server v5.35. Running this command is strongly recommended since search results for past file contents may be incomplete. If this command isn't run, users can search older files based on filename only.
+    Extracts and indexes the contents of files shared prior to upgrading to Mattermost Server v5.35. Running this command is strongly recommended since search results for past file contents may be incomplete. If this command isn't run, users can search older files based on filename only. 
     
     If you're using `Elasticsearch <https://docs.mattermost.com/deployment/elasticsearch.html>`__ search, you must rebuild the search index after running the content extraction command.
     
-    If you're using `Bleve <https://docs.mattermost.com/deployment/bleve.html>`__ search, you must disable Bleve before running the content extraction command. Once extraction is complete, re-enable Bleve, then rebuild the search index.
+    If you're using `Bleve <https://docs.mattermost.com/deployment/bleve.html>`__ search, you must disable Bleve before running the content extraction command. Once extraction is complete, re-enable Bleve, then rebuild the search index. The content extraction command doesn't run on index files located somewhere other than ``/opt/mattermost``, such as ``/opt/mattermost/index``. 
```

Comment:
I don't think this is the real problem underneath.

Reply:
Sounds good, @jespino. Based on your investigation into this issue, please let me know if clarifications to the product documentation is needed. I'm going to close this PR; we can open a new one if needed.